src/main/java/com/epam/prd/jjop/helpers/objects/NullWonder.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.prd.jjop.helpers.objects;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Objects;
+
+/**
+ * Description of @see java.util.Objects regarding null problem solving
```

Comment:
```suggestion
 * Description of {@link java.util.Objects} regarding null problem solving
```

`@see` can only be used at the end of javadoc.